module/VuFind/tests/unit-tests/src/VuFindTest/RecordTab/HoldingsILSTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * RecordTab Manager Test Class
```

Comment:
You should update the comment to actually describe the class. :-)

Reply:
This should be done as of `914397652e`, though it's reluctant to update here for some reason.

Reply:
Yes, I'm seeing the changes. Maybe you are looking at a link to a specific old commit, but if you go to the files tab of the PR, it should be up to date.